docs/unified-test-documentation/dasharo-compatibility/315-network-boot.md

```diff
@@ -234,3 +234,72 @@ server.
 1. The iPXE application boots successfully.
 1. iPXE obtains an IP address.
 1. iPXE boots an `Debian 11` from netboot.xyz.
+
+## USB008.001 Enable Network Boot (Firmware)
```

Comment:
@PrzemyslawBanasiak13 why the tag for this test is `USB`?

Reply:
fixed